in_text/text_02_drivers.yml

```diff
@@ -0,0 +1,731 @@
+title: >-
+  Data to support network-wide 7-day ahead forecasting of water temperature in the Delaware River Basin: 2) model driver data 
+
+abstract: >-
+  <p>Observations related to water and thermal budgets in the Delaware River Basin. Data from reservoirs in the basin include
+  reservoir characteristics (e.g., bathymetry), daily water levels, daily depth-resolved water temperature observations, 
+  and daily inflows, diversions, and releases. Data from streams in the basin include daily flow and temperature observations.  
+  Data were compiled from a variety of sources to cover the modeling period (1980-2022), including the National Water Inventory System, 
+  Water Quality Portal, EcoSHEDS stream water temperature database, ReaLSAT, and the New York Department of Environmental Conservation.
+  The data are formatted as a single csv (comma separated values) or zipped csv. 
+  
+  <p For modeling purposes, we sought to create a test set of flow and temperature observations that were representative of 
+  dynamics throughout the Delaware River basin from water year 1980-present. Test holdouts are documented in the 
+  flow and temperature files. To minimize the possibility of the correlation 
+  between sites and temporal autocorrelation at single sites causing artificially high test performance, we created temporal 
+  holdouts (time periods where data from all sites were reserved for testing), and spatial holdouts (sites where all data were 
+  reserved for testing). In all, this resulted in a train/test split of 66.2%/33.8% for observed temperature reach days, 
+  and 71.4%/28.6% for observed flow reach days. 
+  
+  # <p> For temporal holdouts: All data in the water years 1980-84, 2011-15, and 2021 were reserved for the test set. 
```

Comment:
@limnoliver this paragraph on temporal holdout seems not relevant here. I commented it out for now. can we remove?

Reply:
most of this text is not relevant. I removed and described driver data